rmb-sdk-go/peer/peer.go

```diff
@@ -468,6 +477,10 @@ func (d *Peer) makeEnvelope(id string, dest uint32, session *string, cmd *string
 		Relays: d.relays,
 	}
 
+	if tags != "" {
```

Comment:
shouldn't we validate them?

Reply:
It depends on how strict you want to be
Envelope tags are just hints passed along with the message. The relay may act on some well-known tags (e.g. fail-fast), but apps built on top of the SDK may want to define stricter policies of their own.
SDK: keep minimal, non-opinionated (normalize) -> App layer: free to add stricter validation/authorization via router middleware or request handlers